include/tvm/target/target_kind.h

```diff
@@ -470,7 +470,8 @@ constexpr const char* kRelayToTIR = "RelayToTIR";
           .add_attr_option<String>("model")                       \
           .add_attr_option<Array<String>>("libs")                 \
           .add_attr_option<Target>("host")                        \
-          .add_attr_option<Integer>("from_device")
+          .add_attr_option<Integer>("from_device")                \
+          .add_attr_option<Integer>("target_device_type")
```

Comment:
wondering if we should be producing this from e.g. processing `-mcpu` rather than allowing it to be explicitly set by the user?

Reply:
TargetKind doesn't have "mcpu", so it would be limited to specific kinds.